postgres.Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM postgres
 
 RUN apt-get update && apt-get install -y git-core
 RUN git clone --depth=1 https://github.com/alphagov/verify-event-system-database-scripts.git db-repo \
-  && cp db-repo/migrations/*.sql /docker-entrypoint-initdb.d/
+  && cp db-repo/migrations/*.sql /docker-entrypoint-initdb.d/ \
+  && cp db-repo/test-initialise-scripts/*.sql /docker-entrypoint-initdb.d/
```

Comment:
Part of me really wants the initialise scripts to be listed first... The other part of me has to remind the first part that it just doesn't matter...